examples/open_loop_control_fetch/actions_default_map.csv

```diff
@@ -0,0 +1,421 @@
+joint.wheel_joint_r,joint.wheel_joint_l,joint.arm_joint_0,joint.arm_joint_1,joint.arm_joint_2,joint.arm_joint_3,joint.arm_joint_4,joint.arm_joint_5,joint.arm_joint_6,joint.gripper_finger_joint_r,joint.gripper_finger_joint_l,joint.head_pan_joint,joint.head_tilt_joint
```

Comment:
rename to `actions_starter_content_0000.csv` because that is the new name of this scene (it isn't the default scene in all cases any more because sometimes we use `kujiale_0000` as the default, so I didn't like the name `default`)